erigon-lib/state/aggregator.go

```diff
@@ -31,10 +31,11 @@ import (
 	"sync/atomic"
 	"time"
 
+	rand2 "math/rand/v2"
```

Comment:
addressed "soon to be deprecated warning"

Reply:
crypto/rand?

Reply:
https://pkg.go.dev/golang.org/x/exp/rand -- says to use math/rand/v2 now